AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,9 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+bo.de#$##footer-consent { display: none !important; }
```

Comment:
Both rules can merged to one with selector:

`div[id*="consent"]`

Reply:
Yeah.